clients/drcachesim/scheduler/scheduler.h

```diff
@@ -522,6 +522,17 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * blocking and trigger a context switch.
          */
         uint64_t blocking_switch_threshold = 100;
+        /**
+         * Controls the amount of time inputs are considered blocked at a syscall whose
+         * latency exceeds #syscall_switch_threshold or #blocking_switch_threshold.  A
+         * "block time factor" is computed from the syscall latency divided by either
+         * #syscall_switch_threshold or #blocking_switch_threshold.  This factor is
+         * multiplied by this field #block_time_scale.  For #QUANTUM_TIME, that amount of
+         * time, as reported by the time parameter to next_record(), must pass before the
+         * input is no longer considered blocked.  For instruction quanta, that count of
```

Comment:
Do you instead mean scheduler selections equal to the instruction_quanta times the factor must occur?

Reply:
Yes, "that count" is the final number referenced at line 530 "This factor is multiplied by this field block_time_scale".  Reworded to call it "a final value" and s/that count/that final value/ so it's clearer.